frappe/public/js/frappe/form/controls/int.js

```diff
@@ -10,12 +10,11 @@ frappe.ui.form.ControlInt = class ControlInt extends frappe.ui.form.ControlData
 		super.make_input();
 		this.$input
 			// .addClass("text-right")
-			.on("focus", function () {
+			.on("focusout", function () {
 				setTimeout(function () {
 					if (!document.activeElement) return;
 					document.activeElement.value
 						= me.validate(document.activeElement.value);
-					document.activeElement.select();
 				}, 100);
 				return false;
 			});
```

Comment:
@hasnain2808 validate will be called on change event as well right?

this seems redundant... am I missing something?